implicit.xsd

```diff
@@ -2822,6 +2822,64 @@
 		</xs:choice>
 	</xs:complexType>
 
+	<xs:group name="BasicNodeTypes">
+		<xs:choice>
+			<!-- nodes -->
+			<xs:element name="addition" type="CT_Addition" />
```

Comment:
Better use the the "ref"clause since all those names are now defined as elements at the bottom. 

For example:

<xs:element ref="subtraction"/>
<xs:element ref="multiplication"/>
...